src/Pkg.jl

```diff
@@ -539,9 +539,8 @@ The logic for what path is activated is as follows:
     activate the environment at the tracked path.
   * Otherwise, `s` is interpreted as a non-existing path, which is then activated.
 
-If no argument is given to `activate`, then activate the home project.
-The home project is specified by either the `--project` command line option to
-the julia executable, or the `JULIA_PROJECT` environment variable.
+If no argument is given to `activate`, then use `LOAD_PATH` to determine the
+project to activate.
```

Comment:
> then use `LOAD_PATH` to determine the project to activate

This might be a little to vague. Can we be more specific? 
Perhaps this?

```suggestion
If no argument is given to `activate`, then use the first usable project in `LOAD_PATH`.
```

Reply:
Sure, although I don’t see how this could be understood to mean anything else. The “use […] usable” doesn’t sound great to me, so maybe like this?

> If no argument is given to `activate`, then use the first project found in `LOAD_PATH`.

Reply:
Yeah that sounds good